src/components/processes-configuration.js

```diff
@@ -98,35 +98,27 @@ const ProcessTsos = ({
         })
     );
 
-    useEffect(() => {
+    const updateProcessTsos = (processTsos) => {
         handleProcessTsosChanged(processIndex, processTsos);
-        // Do not add handleProcessTsosChanged as dependency to avoid infinite loop
-        // To be changed
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [processTsos]);
+        setProcessTsos(processTsos);
```

Comment:
why do you `setProcessTsos` here ? In all cases where this is called, it sets the same object as before, so it has no effect ?